marklogic-data-hub/src/test/java/com/marklogic/hub/impl/ClearUserModulesTest.java

```diff
@@ -0,0 +1,126 @@
+package com.marklogic.hub.impl;
+
+import com.fasterxml.jackson.databind.JsonNode;
+import com.marklogic.client.admin.QueryOptionsManager;
+import com.marklogic.client.admin.ResourceExtensionsManager;
+import com.marklogic.client.admin.ServerConfigurationManager;
+import com.marklogic.client.admin.TransformExtensionsManager;
+import com.marklogic.client.io.JacksonHandle;
+import com.marklogic.client.io.QueryOptionsListHandle;
+import com.marklogic.hub.AbstractHubCoreTest;
+
+import java.util.Collection;
+import java.util.List;
+import java.util.Map;
+
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Assumptions;
+import org.junit.jupiter.api.Test;
+
+class ClearUserModulesTest extends AbstractHubCoreTest {
+
+    @Test
+    void testClearUserModules() {
+        new DataHubImpl(runAsDataHubDeveloper()).clearUserModules();
+        Assumptions.assumeTrue(isVersionCompatibleWith520Roles());
+        ModuleCounts beforeClearingModules = new ModuleCounts();
+        beforeClearingModules.setModuleCounts();
+
+        Assertions.assertEquals(1, beforeClearingModules.optionsCount);
```

Comment:
I don't think we want precise counts here, because these assertions will break whenever a REST extension is added/removed (most likely removed). I think all we care about is capturing the initial counts. No need to assert on the expected values.